bin/supply

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+echo "-----> VulsBuildpack/supply"
+
+BIN_DIR=$(cd $(dirname $0); pwd)
+ROOT_DIR=$(dirname $BIN_DIR)
+BUILD_DIR=$1
+CACHE_DIR=$2
+DEPS_DIR=$3
+INDEX=$4
+
+VULS_URL="${VULS_URL:-https://github.com/future-architect/vuls/releases/download/v0.9.6/vuls_0.9.6_linux_amd64.tar.gz}"
```

Comment:
make `0.9.6` an externalizable variable at some point in the future

Reply:
So the point of the `VULS_URL="${VULS_URL:-https://sensible-default/path/to/download}` is actually to externalize the whole URL. I found out from experimenting with using the `latest` tag that this does not work well. One of my issues now is references another variable to the right of the `:-` and it being weird, but I can definitely try that in a bit.

Reply:
I parameterized it anyway.